Formula/hashcat.rb

```diff
@@ -23,6 +23,12 @@ class Hashcat < Formula
 
   depends_on "gnu-sed" => :build
 
+  on_linux do
+    depends_on "opencl-headers" => :build
+    depends_on "ocl-icd"
+    depends_on "pocl"
+  end
+
   def install
     system "make", "CC=#{ENV.cc}", "PREFIX=#{prefix}"
```

Comment:
Similar to other comment, can we try using our copies of some dependencies, at least similar to ArchLinux https://github.com/archlinux/svntogit-community/blob/packages/hashcat/trunk/PKGBUILD#L28-L30
```
    USE_SYSTEM_XXHASH=1 \
    USE_SYSTEM_OPENCL=1 \
    USE_SYSTEM_ZLIB=1
```